File: practice_java_program/notpad/DupliandNonDuplicateElements.java
import java.util.*;
public class DupliandNonDuplicateElements{
	public static void main(String[] args){
		int[] numbers = {1,2,3,4,5,5,6,6,2};
		HashSet<Integer> unique = new HashSet<>();
		HashSet<Integer> duplicate = new HashSet<>();
		for(int num:numbers){
			if(unique.contains(num)){
				duplicate.add(num);
			}
			else{
				unique.add(num);
			}	
		}
		System.out.println("unique element = "+unique);
		System.out.println("duplicate element = "+duplicate);
	}
}

File: python_oriented_interview/notpad/DupliandNonDuplicateElements.py
class DupliandNonDuplicateElements:
	number=[1,2,3,4,5,5,6,6,2]
	uniq=set()
	dupli=set()
	for num in number:
		if num not in uniq:
			uniq.add(num)
		else:
			dupli.add(num)
	print(uniq)
	print(dupli)